README.md

```diff
@@ -49,12 +111,12 @@ oc new-project orchestrator
 
 ### Install the chart with Orchestrator plugin and Notifications plugin
```

Comment:
this will also install the sonataflow platform in non-ephemeral/persistent mode. is that clear?

Reply:
Clarified. Is it better now?